apps/ae_socket_connector/lib/client_runner.ex

```diff
@@ -8,15 +8,15 @@ defmodule ClientRunner do
 
   def start_link(
         {_pub_key, _priv_key, %SocketConnector.WsConnection{}, _ae_url, _network_id, _role, _jobs,
```

Comment:
Such large tuples as arguments are a sign of a bad API design. But I believe I commented on that before :-)

Reply:
Yep so you did; this must be fixed (for later)